BreadPlayer.Views.UWP/ViewModels/ShellViewModel.cs

```diff
@@ -707,7 +732,12 @@ public void Reset()
                 }
             }
         }
-      
+
+        public static async void SaveSettings(Mediafile mediafile)
+        {
+            await _service.UpdateMediafile(mediafile);
+        }
```

Comment:
Unnecessary. I know you are using this in the TagDialog to save the SkipOnShuffle property but this is not needed for that. When a user will set the toggle on, the property will be changed and saved in memory. You can update the Database when the user closes the dialog. Anyway, this is not needed.